pittgoogle/pubsub.py

```diff
@@ -339,27 +332,25 @@ class Subscription:
         .. code-block:: python
 
             # Topic that the subscription should be connected to
-            topic = pittgoogle.Topic(name="lsst-loop", projectid=pittgoogle.ProjectIds().pittgoogle)
+            topic = pittgoogle.Topic(name="lsst-loop", projectid=pittgoogle.ProjectIds().pittgoogle) # currently contains simulated data only
 
             # Specify filters (Optional)
-            keepDiaObjects = "attributes:diaObject_diaObjectId"
-            filterByNPrevDetections = '''
+            _attribute_filter = "attributes:diaObject_diaObjectId" # messages without this attribute key are filtered out (e.g., sources associated with solar system objects would not have this key)
+            _smt_javascript_udf = ''' # objects with <=20 previous detections are filtered out
```

Comment:
```suggestion
            # objects with <=20 previous detections are filtered out
            _smt_javascript_udf = '''
```

Otherwise the comment will be included in the string.